js/core.js

```diff
@@ -278,4 +278,94 @@ $(document).on("touchstart touchend", function(e) {
       }
     }
   }
-});
\ No newline at end of file
+});
+
+var link_StrayRogue = "<a href=\"https://twitter.com/StrayRogue\" target=\"_blank\">@StrayRogue</a>";
+var link_DitzyFlama = "<a href=\"https://twitter.com/DitzyFlama\" target=\"_blank\">@DitzyFlama</a>";
+var link_EricHuber = "<a href=\"https://erics.site/?utm_source=bongo.cat\" target=\"_blank\">Eric Huber</a> (<a href=\"https://twitter.com/Externalizable\" target=\"_blank\">@Externalizable</a>)";
+var i18n_map = {
+  "Bongos": {
+    "en": "Bongos",
+    "ca": "Bongos"},
+  "Cymbal": {
+    "en": "Cymbal",
+    "ca": "Plat"},
+  "Cowbell": {
+    "en": "Cowbell",
+    "ca": "Esquellot"},
+  "Tambourine": {
+    "en": "Tambourine",
+    "ca": "Pandereta"},
+  "Meow": {
+    "en": "Meow",
+    "ca": "Miol"},
+  "SPACE": {
+    "en": "SPACE",
+    "ca": "ESPAI"},
+  "Piano": {
+    "en": "Piano",
+    "ca": "Piano"},
+  "Marimba": {
+    "en": "Marimba",
+    "ca": "Marimba"},
+  "Keyboard": {
+    "en": "Keyboard",
+    "ca": "Teclat"},
+  "LEFT": {
+    "en": "LEFT",
+    "ca": "ESQUERRA"},
+  "RIGHT": {
+    "en": "RIGHT",
+    "ca": "DRETA"},
+  "MEOW": {
+    "en": "MEOW",
+    "ca": "MIOL"},
+  "other-lang": {
+    "en": "<a href=\"/?lang=ca\">En català</a>",
+    "ca": "<a href=\"/?lang=en\">In English</a>"},
+  "courtesy": {
+    "en": "Art courtesy of "+link_StrayRogue,
+    "ca": "Art per cortesia de "+link_StrayRogue},
+  "meme": {
+    "en": "Meme by "+link_DitzyFlama,
+    "ca": "Meme de "+link_DitzyFlama},
+  "website": {
+    "en": "Website by "+link_EricHuber,
+    "ca": "Lloc web d&#39"+link_EricHuber},
+  "cookies": {
+    "en": "This website uses cookies to analyze traffic via anonymized and aggregated data.",
+    "ca": "Aquest lloc web utilitza galetes per analitzar el trànsit mitjançant dades agregades i anonimitzades."}
```

Comment:
@rhaps0dy Just to double check: are you from Catalonia, or do you have advanced knowledge of the Catalan language? Otherwise it wouldn't hurt to have this proofread for safety.

Reply:
I'm from Catalonia. There is one ambiguity in the translation but it's almost inconsequential*. Still, I'll ask my friends to proofread.

* in Catalan, cats go "mèu", but the sound itself is called a "miol", whereas in English I think these are both "meow". I've opted for the name of the sound, much like the label is "Cowbell" but the cowbell goes "tung!" or something like that.

Reply:
Also, @mrcasals could check 😉

Reply:
Looks good to me! 😄